internal/commands/ip_access/delete.go

```diff
@@ -0,0 +1,60 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// CommandMetaDelete is the command meta for the `ip-access delete` command
+var CommandMetaDelete = cli.CommandMeta{
+	Use:     "delete",
+	Display: "allowed IP delete",
```

Comment:
use `"ip-access delete"` here

---

also need to add `HelpText` and `Description` here to be complete, work with @kpham1231 if need be

Reply:
will create a ticket and a `TODO`